test/e2e/node/container_probe.go

```diff
@@ -33,36 +31,23 @@ import (
 
 const (
 	probTestContainerName       = "test-webserver"
-	probTestInitialDelaySeconds = 30
+	probTestInitialDelaySeconds = 15
```

Comment:
Just to make the test time shorter. Now we are going to run this on node e2e, and at least for now, node e2e is running sequentially.

In the future, we may want to:
1) Make node e2e run parallel
2) Set different timeout for e2e and node e2e.